hera_mc/cm_dataview.py

```diff
@@ -27,149 +25,42 @@ def __init__(self, session=None):
             self.session = db.sessionmaker()
         else:
             self.session = session
-        self.parts_list = None
-        self.fc_map = None
 
-    def print_fully_connected(self, fully_connected):
+    def ants_by_day(self, start, stop, interval, filename, station_types_to_check='default', output_date_format='jd'):
         """
-        Prints out correlator information from 'standard' fully_connected list
+        Return dated list of connected stations.
 
         Parameters:
         ------------
-        fully_connected:  list of fully connected stations provided by the correlator
-        methods in geo_handling
+        start:  start astropy.Time
+        stop:  stop astropy.Time
+        interval:  interval in days
+        filename:  filename to write to.  If stdout it will also return a dictionary
```

Comment:
That was because the approach wasn't linear ;-)
I've changed it to be more sensible:  if run via script you always want a file and to see it (ok, maybe you don't want both but it is slow so always writing the file is prudent and it is a script to printing to the screen is probably harmless).  If called, it only returns the dictionary.